compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -34,6 +35,399 @@ namespace mlir::iree_compiler::IREE::GPU {
 constexpr int64_t kCacheLineSizeBits = 128 * 8;
 constexpr int64_t kPreferredCopyNumBits = 128;
 
+//===----------------------------------------------------------------------===//
+// Workgroup Size Multiples
+//===----------------------------------------------------------------------===//
```

Comment:
I think we can move this into its own file (`TileInferenceUtils.cpp` or something) and put it in common because there isn't anything GPU specific here.